modules/ROOT/pages/_partials/sync/server-metrics-partial.adoc

```diff
@@ -0,0 +1,65 @@
+= Monitoring a {sync-server} Application
+
+== Prerequisites
+
+* [x] You have provisioned the xref:mobile-metrics.adoc[{metrics-service} service].
+* [x] You have provisioned your xref:openshift-working-with-apb.adoc[{sync-server} application]
```

Comment:
This hyperlink is not [displayed correctly](https://deploy-preview-631--aerogear-docs-preview.netlify.com/aerogear/latest/data-sync/#monitoring-a-voyager-server-application) - is it pointing to a document which is not created yet?

Reply:
Fixed now, thanks for catching that!